styles/index.css

```diff
@@ -0,0 +1,11 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Heebo:wght@100..900&display=swap');
```

Comment:
- [ OPTIONAL ] Consider using CSS variables (custom properties) to define colors, font families, and other common values. This can make it easier to update the styling of your website globally.

sample:

``` css
/* Define CSS variables for colors */
:root {
    --primary-color: #21243D;
    --button-background: #FF6464;
    --button-border: #FF6464;
    --button-text: #FFF;
}

/* Rest of your CSS styles */
.title {
    color: var(--primary-color);
    /* ...other styles... */
}

.hero-btn {
    border-radius: 2px;
    background: var(--button-background);
    border: 1px solid var(--button-border);
    color: var(--button-text);
    /* ...other styles... */
}
```

**you can read more about that [here](https://www.freecodecamp.org/news/what-are-css-variables-and-how-to-use-them/)**